lib/fontsGenerators.js

```diff
@@ -8,10 +8,11 @@ const wawoff2 = require("wawoff2");
 
 const { writeFile } = require("./utils");
 
-const generateSVGFont = (iconsData, fontName, dest) => {
+const generateSVGFont = (iconsData, fontName, dest, options) => {
   const fontStream = new SVGIcons2SVGFontStream({
     fontName,
-    normalize: true,
+    fontHeight: options.fontHeight,
```

Comment:
Isn't it risky (or just dirty) to pass `undefined` to `fontHeight` unless someone defined it explicitly?

Reply:
In svgicons2svgfont the fontHeight value is either passed in if undefined, calculated by analyzing all svg files and using the max height found.
https://github.com/nfroidure/svgicons2svgfont/blob/94968febca43b10e6b498a111198d70e1790c1ca/src/index.js#L372